__tests__/App-test.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @format
+ */
+
+import 'react-native';
+import React from 'react';
+import App from '../App';
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer';
+
+it('renders correctly', () => {
+  renderer.create(<App />);
```

Comment:
does this do any assertion?

Reply:
this is autogenerated unit test framework, does not do any assertion